submissions/Zakkarat/memory-pair-game/app.js

```diff
@@ -35,18 +38,18 @@ function cardFlip(elem) {
 
 function checkEqual() {
   if (check[0].src == check[1].src){
-    checking = false;
+    checkSecond = false;
     founded.push(check[0]);
     check = [];
-    setTimeout(function() {checking = true;}, 1500)
+    setTimeout(function() {checkSecond = true;}, 1500)
   }
     if (founded.length == 5) {
       popUp();
   } else {
-    checking = false;
+    checkSecond = false;
```

Comment:
can we use functions here something like setChecking and put paramter here and than use setChecking(false)? (this exemplary code :)

Reply:
Done!